ppr-ui/tests/unit/RegistrationLengthTrustAmendment.spec.ts

```diff
@@ -0,0 +1,216 @@
+// Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { getVuexStore } from '@/store'
+import CompositionApi from '@vue/composition-api'
+import { mount, createLocalVue, Wrapper } from '@vue/test-utils'
+import {
+  LengthTrustIF
+} from '@/interfaces'
+import {
+  mockedRepairersLien,
+  mockedSelectSecurityAgreement,
+  mockedVehicleCollateralExisting,
+  mockedGeneralCollateralExisting,
+  mockedDebtorsExisting,
+  mockedSecuredPartiesExisting
+} from './test-data'
+
+// Components
+import { RegistrationLengthTrustAmendment, EditTrustIndenture } from '@/components/registration'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+// Input field selectors / buttons
+const undoButton: string = '#trust-indenture-undo-btn'
+const amendButton: string = '#trust-indenture-amend-btn'
+const doneButton: string = '#done-btn-trust-indenture'
+const cancelButton: string = '#cancel-btn-trust-indenture'
+
+/**
+ * Returns the last event for a given name, to be used for testing event propagation in response to component changes.
+ *
+ * @param wrapper the wrapper for the component that is being tested.
+ * @param name the name of the event that is to be returned.
+ *
+ * @returns the value of the last named event for the wrapper.
+ */
+function getLastEvent (wrapper: Wrapper<any>, name: string): any {
```

Comment:
please import this from test utils

Reply:
Same comment as Steph: removed as not used.